boards.txt

```diff
@@ -261,6 +261,51 @@ Generic_nRF51822.menu.lfclk.lfrc.build.lfclk_flags=-DUSE_LFRC
 Generic_nRF51822.menu.lfclk.lfsynt=Synthesized
 Generic_nRF51822.menu.lfclk.lfsynt.build.lfclk_flags=-DUSE_LFSYNT
 
+Beacon_PCA20006.name=Nordic Beacon Kit (PCA20006)
+
+Beacon_PCA20006.upload.tool=sandeepmistry:openocd
+Beacon_PCA20006.upload.target=nrf51
+Beacon_PCA20006.upload.maximum_size=262144
+
+Beacon_PCA20006.bootloader.tool=sandeepmistry:openocd
+
+Beacon_PCA20006.build.mcu=cortex-m0
+Beacon_PCA20006.build.f_cpu=16000000
+Beacon_PCA20006.build.board=GENERIC
+Beacon_PCA20006.build.core=nRF5
+Beacon_PCA20006.build.variant=PCA20006
+Beacon_PCA20006.build.variant_system_lib=
+Beacon_PCA20006.build.extra_flags=-DNRF51 -DBOARD_PCA20006 -DNRF51822
```

Comment:
We can remove the `-DBOARD_PCA20006 -DNRF51822` args, the IDE should generate an equivalent for the `BOARD_PCA20006` - see the verbose build logs.